browser/components/MetricsLists.tsx

```diff
@@ -49,95 +49,68 @@ export default class MetricsLists extends React.Component<Props, {}> {
     });
   }
 
-  topContributors = list => {
-    return list ? (
-      list.map(item => {
-        return (
-          <li key={'topContributors_' + item.alias}>
-            {item.alias} | {item.count}
-          </li>
-        );
-      })
-    ) : (
-      <div />
-    );
-  };
-
   render() {
-    const metrics = this.metricsAll();
     const curYear = new Date().getFullYear();
     return (
       <div id="metrics">
         <h2>Metrics</h2>
-        <div className="row">
-          <div className="col-xl-3">
-            <RadzChart
-              data={this.getProcessedList(
-                this.props.topContribProjectsThisYear.slice(0, 10)
-              )}
-              centerText={curYear.toString()}
-              height={200}
-              width={300}
-              id={'viz' + curYear.toString()}
-              cornerRadius={0.3}
-              padAngle={0.015}
-              centerTextSize={'40px'}
-              centerTextdx={'-1em'}
-            />
-          </div>
+        <div>
+          <div className="row">
+            <div className="col">
+              <div className="float-left">
+                <RadzChart
+                  data={this.getProcessedList(
+                    this.props.topContribProjectsThisYear.slice(0, 10)
```

Comment:
Pull items out of props and into const variables in render. That way if we want to change the name of the props in the future we only have to do it in a single location.